packages/core/src/interpreter.ts

```diff
@@ -388,8 +388,8 @@ export class Interpreter<
     return this;
   }
   public subscribe(
-    observer: Observer<
-      State<TContext, TEvent, any, TTypestate, TResolvedTypesMeta>
+    observer: Partial<
```

Comment:
I've rechecked with RxJS and listener types are actually all optional (well, not exactly, but I'm not sure if we want to do [this](https://github.com/ReactiveX/rxjs/blob/9a85e01625a1ee7a23e4ab2c2c6d3089a58e2e54/src/internal/types.ts#L170)).

It was really annoying in tests for me to implement this:
```js
service.subscribe({
  next: noop,
  error: noop,
  complete: completeCb
})
```
and I think that it makes sense to make it easier to just provide a single callback type, so I've followed that instinct and wrapped `Observer` at the parameter position `Partial`